src/instruments/src/EFB/Settings/Pages/RealismPage.tsx

```diff
@@ -121,6 +122,10 @@ export const RealismPage = () => {
                 )}
             </SettingGroup>
 
+            <SettingItem name={t('Settings.Realism.SyncEfis')}>
```

Comment:
This is available in Localazy now.
Please  add "unrealistic" as for Autofill Checklist above.